app/models/contact.rb

```diff
@@ -3,11 +3,11 @@ class Contact < ApplicationRecord
 
   belongs_to :ngo
 
-  validates :first_name, length: { in: 1..50 }
-  validates :last_name, length: { in: 1..50 }
-  validates :email, presence: true
-  validates :phone, presence: true
-  validates :street, presence: true
-  validates :zip, presence: true
-  validates :city, presence: true
+  validates :first_name, length: { in: 1..50 }, on: :create
```

Comment:
Hey @Thrillberg, these validations are actually mandatory - we needs them on update as well. Could you please just revert this one file?